azure-nextgen-py-appservice-docker/__main__.py

```diff
@@ -0,0 +1,100 @@
+import pulumi
+from pulumi_azure_nextgen.containerregistry import latest as containerregistry
+from pulumi_azure_nextgen.resources import latest as resources
+from pulumi_azure_nextgen.storage import latest as storage
+from pulumi_azure_nextgen.web import latest as web
+import pulumi_docker as docker
+import pulumi_random as random
+
+resource_group = resources.ResourceGroup("resourceGroup",
+    resource_group_name="appservice-docker-rg",
+    location="WestUS")
```

Comment:
Should this use the location from the config?

Reply:
Yes... I was certain I did that already... restoring.